brace-initialization-patch

Fix strict warning message treated as an error
(-Werror=missing-braces) by explicitly initializing every struct
member to zero.

diff --git a/libdpe/pe_allocspace.c b/libdpe/pe_allocspace.c
index d30de1f..b813874 100644
--- a/libdpe/pe_allocspace.c
+++ b/libdpe/pe_allocspace.c
@@ -61,7 +61,7 @@ pe_set_image_size(Pe *pe)
 	struct pe32plus_opt_hdr *opthdr = pe->state.pe32plus_exe.opthdr;
 
 	Pe_Scn *scn = NULL;
-	struct section_header shdr = { 0, }, tmp_shdr;
+	struct section_header shdr = {{ 0 }, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, tmp_shdr;
 	if (pehdr->sections < 1)
 		return -1;
 
